debian/rules

```diff
@@ -9,6 +9,8 @@ export DH_GOLANG_EXCLUDES=tests
 export DH_GOLANG_GO_GENERATE=1
 
 export PATH:=${PATH}:${CURDIR}
+# make sure that correct go version is found on trusty
```

Comment:
Nope, trusty's go version is too old and the updated package golang(-*)-1.6 installs to the respective directory. This follows advice given by slangasek.